### ISSUE TITLE
ModuleNotFoundError: No module named 'importlib.metadata' with Python 3.7.7

### ISSUE THREAD

**Issue by rvais** [rh-messaging-qe/yacfg#77]
When the current/latest version of yacfg is installed directly from repository and then used, following error appears:
```
[yacfg](main|*)$ yacfg --list-profiles
Traceback (most recent call last):
  File "/home/rvais/Demo/yacfg/venv3/bin/yacfg", line 2, in <module>
    from yacfg.yacfg_cli import main
 File "/home/rvais/Demo/yacfg/src/yacfg/__init__.py", line 15, in <module>
   import importlib.metadata
ModuleNotFoundError: No module named 'importlib.metadata'
```
### Steps to reproduce
```
git clone git@github.com:rh-messaging-qe/yacfg.git
cd yacfg
python3 -m venv ./venv3
python3 -m venv ./venv3 --upgrade
source ./venv3/bin/activate
pip install --upgrade pip
pip install poetry
poetry install
yacfg --list-profiles
```
### Environment
- yacfg Version: current main branch (commit [9b0e1a9](https://github.com/rh-messaging-qe/yacfg/commit/9b0e1a93aba47d8dc455b8cfd8a8b19597c99b9d))
- python Version: 3.7.7
- Platform: Fedora 31

**Comment by rvais** [comment]
The issue is that built-in library in question has been renamed for python 3.8+ and there is simple work around at least for development purposes. Edit following file and rename used library from `importlib.metadata` to original `importlib_metadata`
`$ vi src/yacfg/__init__.py`

**Comment by enkeys** [comment]
It's already fixed in develop 0.10.0 branch, pls wait for the release. Thank you.

**Comment by rvais** [comment]
It's not something I intend to commit, however I need that workaround so I can work the profile I will need to use for demo purposes.
